tracing-subscriber/src/registry/mod.rs

```diff
@@ -68,8 +68,7 @@ use tracing_core::{field::FieldSet, span::Id, Metadata};
 mod extensions;
 #[cfg(feature = "registry")]
 mod sharded;
-#[cfg(feature = "registry")]
-mod stack;
+pub(crate) mod stack;
```

Comment:
@hawkw this was in your branch, but don't think it's intended. can you confirm?

Reply:
whoops, that was an accident. thanks for noticing!

Reply:
fixed!